omero/developers/Web/CreateApp.txt

```diff
@@ -323,76 +323,8 @@ Users can then configure this on the command line as follows:
 
     $ bin/omero config set omero.web.yourapp.foo '{"userkey": "userval"}'
 
+Linking From Webclient
```

Comment:
Subheadings should only have an initial capital to match the rest of the page - 'Linking from webclient'